src/tokens/solana.tokenlist.json

```diff
@@ -153401,4 +153409,4 @@
     "minor": 3,
     "patch": 3
   }
-}
```

Comment:
Line 153404 contains a } that was deleted when adding lines 153370 through 153377 before saving the changes to this file. When the deleted } was identified when previewing the new lines added and line 153404 indicated that a change was made to the original code text in the file; the } was added back to the file before submitting the commit and requesting a pull request. Hopefully this aid in the review of the pull request submitted and the additional lines 153370 through 153377 can be successfully merged with the other pull requests along with the original main net file by the reviewing party.